Arm/Syntax.lean

```diff
@@ -10,10 +10,10 @@ import Arm.Memory.Separate
 
 namespace ArmStateNotation
 
-/-! We build a notation for `read_mem_bytes $n $base $s` as `$s[$base, $n]` -/
+/-! We build a notation for `$s.mem.read_bytes $n $base $s` as `$s[$base, $n]` -/
```

Comment:
This means that `read/write_mem_bytes` can be thrown away, right, because a user will essentially type in `read/write_bytes` whenever they use this notation?

Reply:
If that's the case, let's just throw them away instead of adding to the confusion

Reply:
Yes, except that `read_mem_bytes` is currently not directly defined in term of `Memory.read_bytes`, so I didn't want to touch that any more than necessary. Let's punt this change to a follow-up PR?